src/model/control_desk.py

```diff
@@ -37,6 +38,10 @@ def __init__(self, uid: str | None = None) -> None:
         self._upper_text = ""
         self.data_changed_callback = None
 
+    def set_pushed_to_device(self) -> None:
+        """set Column pushed to a device"""
```

Comment:
`"""Mark column as synced with device"""`

This method should be private as only the synchronization methods should have access to it.